sources/packages/backend/apps/queue-consumers/src/processors/schedulers/notification/process-notifications.scheduler.ts

```diff
@@ -48,21 +36,41 @@ export class ProcessNotificationScheduler extends BaseScheduler<ProcessNotificat
   /**
    * Process all the unsent notifications and return
    * summary of processing.
-   * @param job process notification job.
+   * @param job process job.
+   * @param processSummary process summary for logging.
    * @returns processing summary.
    */
-  @Process()
-  async processNotifications(
+  protected async process(
     job: Job<ProcessNotificationsQueueInDTO>,
-  ): Promise<QueueProcessSummaryResult> {
+    processSummary: ProcessSummary,
+  ): Promise<string> {
     const processNotificationResponse =
       await this.notificationService.processUnsentNotifications(
         job.data.pollingRecordsLimit,
       );
-    const processSummaryResult: string[] = [
-      `Total notifications processed ${processNotificationResponse.notificationsProcessed}`,
-      `Total notifications successfully processed ${processNotificationResponse.notificationsSuccessfullyProcessed}`,
-    ];
-    return { summary: processSummaryResult } as QueueProcessSummaryResult;
+    processSummary.info(
+      `Total notifications processed ${processNotificationResponse.notificationsProcessed}.`,
+    );
+    processSummary.info(
+      `Total notifications successfully processed ${processNotificationResponse.notificationsSuccessfullyProcessed}.`,
+    );
```

Comment:
Minor. As this is a result, it can be part of return. As there is no E2E tests for this scheduler, there won't be further impacts in changing the return.

Reply:
Added the result to the summary and result also.